### ISSUE TITLE
Error [7069]:The target session is incompatible with the current session in 10.0.17763.437

### ISSUE THREAD

**Issue by manprost** [stascorp/rdpwrap#787]
Hi.

I finally made multiple sessions per user (i.e. _Single session per user_ box not checked) work in 10.0.17763.437 with these settings:

```
[10.0.17763.437-SLInit]
bInitialized.x86 =CD798
bServerSku.x86 =CD79C
lMaxUserSessions.x86 =CD7A0
bAppServerAllowed.x86 =CD7A8
bRemoteConnAllowed.x86=CD7AC
bMultimonAllowed.x86 =CD7B0
ulMaxDebugSessions.x86=CD7B4
bFUSEnabled.x86 =CD7B8
bInitialized.x64 =ECAB0
bServerSku.x64 =ECAB4
lMaxUserSessions.x64 =ECAB8
bAppServerAllowed.x64 =ECAC0
bRemoteConnAllowed.x64=ECAC4
bMultimonAllowed.x64 =ECAC8
ulMaxDebugSessions.x64=ECACC
bFUSEnabled.x64 =ECAD0

[10.0.17763.437]
LocalOnlyPatch.x64=1
LocalOnlyOffset.x64=77A41
LocalOnlyCode.x64=jmpshort
SingleUserPatch.x64=1
SingleUserOffset.x64=3E520
SingleUserCode.x64=Zero
DefPolicyPatch.x64=1
DefPolicyOffset.x64=18025
DefPolicyCode.x64=CDefPolicy_Query_eax_rcx
SLInitHook.x64=1
SLInitOffset.x64=1ACDC
SLInitFunc.x64=New_CSLQuery_Initialize
```

So, if I have the console session opened with user "admin" and I connect to the host via RDP with user "admin", another session gets created, as expected.

But, if I try to issue a "tscon console" command in this new session, I obtain the following error:

> Could not connect sessionID 1 to this session, Error code 7069
> Error [7069]:The target session is incompatible with the current session.

This used to work in previous versions and it was really handy, as it allows to keep _Single session per user_ off but still take control of the console session when it's needed.

Am I doing something wrong?

Thank you.

**Comment by lehha** [comment]
Yes, last update broke a connection to console session. Im wait solution too...

**Comment by manprost** [comment]
Thank you for confirming the bug, let's hope something can be done about it.

**Comment by flesniak** [comment]
Had the same problem. I got it running using the rdpwrap.ini with slightly different definitions for 10.0.17763.437 from this fork: https://github.com/DrDrrae/rdpwrap/blob/master/res/rdpwrap.ini

**Comment by JaWaRo** [comment]
Had the same problem. I got it running using #795

**Comment by manprost** [comment]
Thank you for your answers, guys, and sorry for the late response.

Tried with your settings and, unfortunately, they didn't work for me, they don't allow me to properly open multiple sessions with the same user. I see that the only difference is that my settings have the line:

`SingleUserOffset.x64=3E520`

while yours have the line:

`SingleUserOffset.x64=1322C`

If you have 10.0.17763.437 x64 working with multiple sessions per user and tscon with those settings I guess there's something wrong with my configuration that I'm unable to pinpoint ☹ (I tried in a clean VM and I obtained the same behavior).

Thank you.
